dev-php/symfony-polyfill-mbstring/symfony-polyfill-mbstring-1.28.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Symfony polyfill for the Mbstring extension"
+HOMEPAGE="https://github.com/symfony/polyfill-mbstring"
+SRC_URI="https://github.com/symfony/polyfill-mbstring/archive/v${PV}.tar.gz
+	-> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+BDEPEND="dev-php/theseer-Autoload"
+
+RDEPEND="dev-lang/php:*
+	dev-php/fedora-autoloader"
+
+S="${WORKDIR}/polyfill-mbstring-${PV}"
+
+src_prepare() {
+	default
+
+	phpab \
+		--quiet \
+		--output autoload.php \
+		--template fedora2 \
+		--basedir . \
+		. \
+		|| die
+	echo "\Fedora\Autoloader\Dependencies::required([__DIR__ . '/bootstrap.php']);" >> autoload.php
+}
+
+# TODO: Check if README.md is installed twice
```

Comment:
do you want me to check that? Or is it left over?
As this schmea with einstalldocs is repeated in other ebuilds, it might be you checked it elsewhere?